### ISSUE TITLE
GraphQL tooltip is wrong

### ISSUE THREAD

**Issue by Emily-Jiang** [eclipse/microprofile-starter#384]
On test-start.microprofile.io, when you hover over graphql, you will see the link to the GraphQL release. However, the link includes %s, which was not substituted.

**Comment by Karm** [comment]
@Emily-Jiang The fix is this method:

```
    public String getSpecificationStandaloneLink(StandaloneMPSpec spec) {
        return String.format(spec.tagURL, spec.version);
    }
```

I'm gonna push some refactoring, painting the bikeshed and this fix....

**Comment by Karm** [comment]
Fixed by #386
